graylog2-web-interface/src/views/components/widgets/EditMessageList.jsx

```diff
@@ -44,12 +53,12 @@ const EditMessageList = ({ children, config, fields, onChange }: Props) => {
   const onDecoratorsChange = newDecorators => onChange(config.toBuilder().decorators(newDecorators).build());
 
   return (
-    <Row>
-      <Col md={3}>
+    <Row style={{ height: '100%', paddingBottom: '15px' }}>
```

Comment:
I could not use a styled component here, due to some more complex flow errors.

Reply:
Let us talk about the flow error at some point and see what we can do. Merging this nonetheless.